include/deal.II/grid/reference_cell.h

```diff
@@ -34,13 +34,9 @@ template <int dim, int spacedim>
 class Mapping;
 template <int dim>
 class Quadrature;
-template <int dim>
-class ScalarPolynomialsBase;
 #endif
 
-/**
- * A namespace for reference cells.
- */
+
 class ReferenceCell;
```

Comment:
This should probably also be hidden from `doxygen`.

Reply:
Since it tests ok and I'd like to clear my backlog of patches, would you allow me to do this in a follow-up?

Reply:
Sure.